src/layout/GenericComponent.tsx

```diff
@@ -117,6 +117,14 @@ export function GenericComponent<Type extends ComponentTypes = ComponentTypes>({
     shallowEqual,
   );
 
+  //Hide validation message if component maxLength exists and exceeded (handled by component's own message) and it's the only error
+  const componentMaxLength = node.item?.maxLength;
+  const hideValidationMessage =
+    componentMaxLength &&
+    formData?.simpleBinding &&
+    formData?.simpleBinding?.length > componentMaxLength &&
+    componentValidations?.simpleBinding?.errors?.length === 1;
```

Comment:
I agree - we could remove this check (or make it more robust? It could possibly check if the one error message is the one for maxLength in the model as well)

Reply:
Yeah that is a good point 👍 I pushed an update where the validation messages are filtered for maxLength-messages. This seems to work smooth. 
I'm not sure, but it doesn't look like app devs can customize this validation text. However, if it's possible to customize or we plan it in the future, I found a (hacky?) way to check if maxLength in datamodel exists and its value.
I left it out for now, because I'm unsure if this is "smør på flesk". I can implement it if you think it's a good idea.

```
const schema = useAppSelector((state) => state.formDataModel.schemas);
const flatSchema = Object.assign({}, ...Object.values(schema));
const bindingValue = (node?.item?.dataModelBindings as { simpleBinding?: string })?.simpleBinding;
const schemaKey = bindingValue?.split('.').slice(-2, -1)[0];
const schemaMaxLength= schemaKey && flatSchema.definitions?.[schemaKey]?.properties.value?.maxLength;
```

_note: the tests runs well in local, I just need to update tt02._

Reply:
Ah, naah - I don't think that's all that reliable - not all schemas are even built using `definitions`, and looking up in the Schema is unreliable even in `useBindingSchema()` that uses a 3rd party tool that properly parses the schema to do it. Otherwise this looks good! 👍